core/serverutil.go

```diff
@@ -158,3 +160,34 @@ func workflowToRoleTree(root workflow.Role) (ri *pb.RoleInfo) {
 	}
 	return
 }
+
+
+type Connections struct {
```

Comment:
Is this part of a channel multiplexer? I feel it could use some inline docs or a more specific name.